backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/types/codec/impl/ElasticsearchLocalTimeFieldCodec.java

```diff
@@ -19,4 +22,9 @@ public ElasticsearchLocalTimeFieldCodec(DateTimeFormatter delegate) {
 	protected LocalTime nullUnsafeParse(String stringValue) {
 		return LocalTime.parse( stringValue, formatter );
 	}
+
+	@Override
+	protected Long nullUnsafeScalar(LocalTime value) {
+		return value.atDate( LocalDate.of( 1970, Month.JANUARY, 1 ) ).atZone( ZoneOffset.UTC ).toInstant().toEpochMilli();
```

Comment:
You can just call `value.toInstant( ZoneOffset.UTC )` instead of `atZone(...).toInstant()`.

Also, `LocalDate.of( 1970, Month.JANUARY, 1 )` should be a constant.

Also, I think `value.toNanoOfDay() / 1_000_000` would achieve the same result, but maybe that would be less readable.

Reply:
OK for the first and the second suggestions. For the last one I would prefer to keep `toEpochMilli()`. Yes, for readability reasons.